### PR TITLE
JUMP_BACKWARD with quickening

### PR DESCRIPTION
**Pull request by iritkatriel**
This is the same as https://github.com/python/cpython/pull/32115, but with quickening put back in. It doesn't work.

### CODE REVIEW

File: Python/ceval.c [hunk 1]
```diff
@@ -4069,8 +4069,8 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
             DISPATCH();
         }
 
-        TARGET(JUMP_ABSOLUTE_QUICK) {
-            PREDICTED(JUMP_ABSOLUTE_QUICK);
+        TARGET(JUMP_BACKWARD_QUICK) {
+            PREDICTED(JUMP_BACKWARD_QUICK);
             assert(oparg < INSTR_OFFSET());
             JUMPTO(oparg);
```
Comment (brandtbucher) [inline Python/ceval.c]:
```suggestion
            JUMPBY(-oparg);
```

**Review by brandtbucher — CHANGES_REQUESTED** [review]
Found it!

File: Python/ceval.c (hunk 1 above)
Comment (iritkatriel) [reply Python/ceval.c]: Ah!

**Comment by iritkatriel** [comment]
The tests pass now, so it was my silly mistake. I'll close this, and we can decide what to do about https://github.com/python/cpython/pull/32115 (whether to commit as is or to put back the quickening as here).

**Pull request closed without merging**